core/src/test/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitorTest.java

```diff
@@ -363,7 +477,13 @@ private void stubQueries(StubbedQuery... queries) {
       assertThat(nextQuery.queryString).isEqualTo(queryString);
       assertThat(nextQuery.parameters).isEqualTo(parameters);
       if (nextQuery.error) {
-        new CompletableFuture<AdminResult>().completeExceptionally(new Exception("PlaceHolder"));
+        CompletableFuture<AdminResult> future = new CompletableFuture<>();
```

Comment:
I think this was a bug: the future wasn't being returned to the caller.

Reply:
nit: this could be shortened with `CompletableFutures.failedFuture`.